### ISSUE TITLE
[Entrega 2] Uso código anterior

### ISSUE THREAD

**Issue by fedezang** [RodrigoToroIcarte/IIC2113-2023-1#18]
Hola!
Para esta entrega ocupamos el nuevo código base pero hay que agregar los cambios de la entrega anterior cierto?

**Comment by RodrigoToroIcarte** [comment]
Sí, esa es la idea. También puedes partir desde cero si quieres. No estás atado a tu entrega anterior :)
